apps/cnquery/cmd/login.go

```diff
@@ -200,9 +200,6 @@ func register(token string, annotations map[string]string, timer int, splay int,
 		}
 
 		log.Debug().Msg("store configuration")
-		// overwrite force, otherwise it will be stored
-		viper.Set("force", false)
```

Comment:
This never worked. And it was trying to unset a config setting in a single place. Moved it to the storing of the config instead.